Cargo.toml

```diff
@@ -16,6 +16,10 @@ winapi = { version = "0.3.8", features = ["combaseapi", "libloaderapi"] }
 widestring = "0.4.0"
 # com = "0.1"
 com = { git = "https://github.com/microsoft/com-rs.git" }
+memory-module-sys = { version = "0.3.0" }
 
 [dev-dependencies]
-winit = "0.20.0"
+winit = "0.22.0"
+# futures = "0.3.4"
+# tungstenite = "0.10.1"
+ws = { git = "https://github.com/peddermaster2/ws-rs.git" }
```

Comment:
Were you experimenting with some other example but forgot to revert the changes to `dev-dependencies`?

Reply:
Yes sorry i have some test for other thing and i don't remove in this pull :s